packages/server-wallet/src/models/channel.ts

```diff
@@ -343,6 +343,21 @@ export class Channel extends Model implements RequiredColumns {
     return !!this.assetHolderAddress;
   }
 
+  public get isAppChannel(): boolean {
+    return !this.isLedger;
+  }
+
+  public get isRunning(): boolean {
+    // running if:
+    //  1. the supported state implies a post-fund-setup
+    //  2. no isFinal states exist
+
+    const havePostFund = !!this.supported && this.supported.turnNum >= 2 * this.nParticipants() - 1;
+    const noFinalStates = _.every(this.sortedStates, s => !s.isFinal);
```

Comment:
There shouldn't be more than `numParticipants` states in normal behaviour, since the wallet removes states that are older than those in the support of the latest supported state.